### PR TITLE
Completed the documentation for Gaussian Mixture Model

### PR DESCRIPTION
**Pull request by vinc3nati**
I have completed the documentation of Gaussian Mixture Model for issue #142 .

### CODE REVIEW

File: Gaussian Mixture Model/README.md [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+
+_More about log-likelihood [here](https://en.wikipedia.org/wiki/Likelihood_function#:~:text=Log%2Dlikelihood%20function%20is%20a,to%20maximizing%20the%20log%2Dlikelihood.)_
+
+\
+<space>
+**References :**
+
+1. [Elements of Statistical Learning](https://web.stanford.edu/~hastie/ElemStatLearn/)
+
+2. [Scikit Learn](hhttp://scikit-learn.org/stable/modules/mixture.html#:~:text=A%20Gaussian%20mixture%20model%20is,Gaussian%20distributions%20with%20unknown%20parameters.)
```
Comment (geekquad) [inline Gaussian Mixture Model/README.md:200]: Link not working. 
Please make the necessary changes.
image

**Review by geekquad — CHANGES_REQUESTED** [review]
Please make the necessary changes.

**Comment by vinc3nati** [comment]
> Please make the necessary changes.

Roger that

**Comment by vinc3nati** [comment]
Done sir @geekquad

**Review by geekquad — APPROVED** [review]
Amazing work @vinc3nati.

**Pull request merged**